.github/workflows/scheduled-tests.yml

```diff
@@ -56,7 +56,7 @@ jobs:
     - name: Test with plugin
       run: |
         source .venv/bin/activate
-        pytest --expensive-tests
+        pytest --expensive-tests --timeout 30
```

Comment:
I think the timeout that is getting us is from [this decorator](https://github.com/mlcommons/newhelm/blob/2d7b1215db4f8384398d0a04dd6b94bb1e8643c9/plugins/validation_tests/test_object_creation.py#L50). If you agree, I think we should change the decorator instead.

Alternatively, maybe specific Safe tests have gotten big enough that we should add them to the TOO_SLOW list.

Reply:
My mistake. Thanks for that. I feel like you'll have better insight into if these make sense to keep in. I updated 10 to 30 seconds in the decorator, but totally cool if you'd rather just remove the coverage for them.